modules/apps/object/object-test/src/testFunctional/tests/ObjectPersonalData.testcase

```diff
@@ -22,6 +22,273 @@ definition {
+				labelName = "Custom Object 186468",
+				objectName = "CustomObject186468",
+				pluralLabelName = "Custom Objects 186468");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Field",
+				fieldName = "customObjectField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "CustomObject186468");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject186468");
+
+			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+				fieldName = "customObjectField",
+				objectName = "CustomObject186468",
+				value = "Entry A");
+		}
+
+		task ("And given a user with permission to add object entries") {
+			JSONUser.addUser(
+				userEmailAddress = "userea@liferay.com",
+				userFirstName = "userfn",
+				userLastName = "userln",
+				userScreenName = "usersn");
+
+			JSONUser.setFirstPassword(
+				agreeToTermsAndAnswerReminderQuery = "true",
+				requireReset = "false",
+				userEmailAddress = "userea@liferay.com");
+
+			JSONRole.assignRoleToUser(
+				roleTitle = "Administrator",
+				userEmailAddress = "userea@liferay.com");
+
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "userea@liferay.com",
+				userLoginFullName = "userfn userln");
+		}
+
+		task ("And given several object entries added by the user") {
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject186468");
+
+			for (var entryName : list "B,C,D,E") {
+				ObjectAdmin.goToCustomObject(objectName = "CustomObject186468");
+
+				ObjectPortlet.addSingleFieldEntryViaUI(entry = "Entry ${entryName}");
+			}
+		}
+
+		task ("When the admin user anonymizes the user's personal data in Users and Organizations") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "test@liferay.com",
+				userLoginFullName = "Test Test");
+
+			ApplicationsMenu.gotoPortlet(
+				category = "Users",
+				panel = "Control Panel",
+				portlet = "Users and Organizations");
+
+			User.deletePersonalDataCP(
+				userFirstName = "userfn",
+				userLastName = "userln");
+
+			GDPR.anonymizeAllEntriesinApplicationCP(
+				applicationItemsCount = 4,
+				applications = "Objects",
+				totalItemsCount = 4);
+		}
+
+		task ("Then the author of the entries created by the user is Anonymous Anonymous") {
+			User.openUsersAdmin();
+
+			User.activateCP(
+				userName = "userfn userln",
+				userScreenName = "usersn");
+
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "userea@liferay.com",
+				userLoginFullName = "userfn userln");
+
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject186468");
+
+			for (var key_order : list "2,3,4,5") {
+				AssertElementPresent(
+					key_entry = "Anonymous Anonymous",
+					key_order = ${key_order},
+					locator1 = "ObjectPortlet#ENTRY_VALUE_BY_POSITION");
+			}
+		}
+	}
+
+<<<<<<< HEAD
+=======
+	@description = "LPS-186467 - Verify that the entries of a object can be deleted"
+	@priority = 5
+	test CanDeleteObjectEntries {
+		property portal.acceptance = "true";
+
```

Comment:
Be careful when fixing SF errors or rebasing, this shouldn't be part of this commit. Please make sure this test is part of your third commit.

Also, don't forget to change the ticket ID to the story ticket.